source/transactions/tests/pin-mongos.yml

```diff
@@ -1,20 +1,17 @@
 # Test that all the operations go to the same mongos.
 #
-# The assertion is implicit: that all the read operations succeed.
-# If the driver does not properly pin to a single mongos then one of the
-# operations in a transaction will eventually be sent to a different mongos,
-# which is unaware of the transaction, and the mongos will return a command
-# error. An example of such an error is:
+# In tests that don't include command-started events the assertion is implicit:
+# that all the read operations succeed. If the driver does not properly pin to
+# a single mongos then one of the operations in a transaction will eventually
+# be sent to a different mongos, which is unaware of the transaction, and the
```

Comment:
Pedantic note: since [server selection uses random selection](https://github.com/mongodb/specifications/blob/master/source/server-selection/server-selection.rst#why-change-from-mongos-high-availablity-ha-to-random-selection), is there a slight chance these tests will all hit the same mongos?

Reply:
Yes that's why it says "eventually". Do you want me to call this out explicitly? When I run these tests without pinning they always fail.

Reply:
No worries. Was mainly asking for my own understanding :)

Fine as-is.